Initial draft 7.0.0-RC1 blog post (#380)
apache/grails-static-website: posts/2025-08-10-grails-7-rc1.md

* fix: #13705 - revert changes to Metadata from micronaut removal @jdaugherty (#14897)

Full Changelog: [v[%priorVersion]...v[%version]](https://github.com/apache/grails-core/compare/v[%priorVersion]...v[%version])

Upgrade instructions are available in the [documentation](https://docs.grails.org/[%version]/guide/single.html#upgrading60x). 

## Dependency Upgrades
In this release, we've upgraded several dependency versions, including but not limited to the following:

* Asset Pipeline 5.0.12 (now cloud.wondrify.asset-pipeline)
* Groovy 4.0.28
* Spring Framework 6.2.9
* Spring Boot 3.5.4
* Gradle 8.14.3
* See all in the [grails-bom](https://docs.grails.org/[%version]/ref/Versions/Grails%20BOM.html).

## Generating a new Grails [%version] application with Grails Forge
Try out Grails today by visiting our online application generator [Grails Forge](https://start.grails.org/).  This is the quickest and the recommended way to get started with Grails.

After installing JetBrains' IntelliJ IDEA 2025.2 or later and the [Grails Plugin](https://plugins.jetbrains.com/plugin/18504-grails), the Grails Application Forge will also be available under New Project in IntelliJ IDEA. 

> Within your newly generated project you can access the Grails CLIs with the grails wrapper

Reviewer: ```suggestion
Within your newly generated project you can access the Grails CLIs with the Grails Wrapper.
```

> See [Types of CLI](https://docs.grails.org/[%version]/guide/gettingStarted.html#downloadingAndInstalling) for details on each CLI

Reviewer: ```suggestion
See the [Types of CLI](https://docs.grails.org/[%version]/guide/gettingStarted.html#_types_of_cli) section in the documentation for details on each CLI.
```

grail-shell-cli

```shell
grailsw
```

grails-forge-cli

```shell
grailsw -t forge
```

## Installing Grails CLIs [%version] with SDKMan

> Alternatively, you can quickly install Grails [%version] CLIs (grails-shell-cli and grails-forge-cli) using [SDKMan](https://sdkman.io/).

Reviewer: ```suggestion
Alternatively, you can quickly install Grails [%version] CLIs (grails-shell-cli and grails-forge-cli) using [SDKMan](https://sdkman.io).
```

> See [Types of CLI](https://docs.grails.org/[%version]/guide/gettingStarted.html#downloadingAndInstalling) for details on each CLI

Reviewer: ```suggestion
See the [Types of CLI](https://docs.grails.org/[%version]/guide/gettingStarted.html#_types_of_cli) section in the documentation for details on each CLI.
```

> 1. If you don't have SDKMan installed, follow the instructions at [SDKMan Installation Guide](https://sdkman.io/install/)
>    to set it up.

Reviewer: ```suggestion
1. If you don't have SDKMan installed, follow the instructions at [SDKMan Installation Guide](https://sdkman.io/install) to set it up.
```

    ```properties
    grailsVersion=[%version]
    ```

3. Make any necessary adjustments to your application code, configuration, and dependencies to ensure compatibility with
   the new version. [See Upgrade Guide](https://docs.grails.org/[%version]/guide/single.html#upgrading60x)

Normally, Grails Core dependencies are automatically updated using the Grails Bill of Materials (BOM). However, if you
have specific versions defined in your build configuration, you may need to manually update them to align with
Grails [%version].

By following these steps, you should be able to transition your existing Grails application to Grails [%version].

### Exploring Alternative Approaches
If manual dependency updates seem daunting, or you want a more streamlined approach, consider the following alternatives:

#### 1. Use Grails Forge Website
Visit [Grails Forge](https://start.grails.org) and generate a new Grails application with Grails [%version]. Compare the
versions in the newly generated application with your existing one to identify any discrepancies. This can serve as a
reference point for your update.

#### 2. Automated Dependency Update Bots

> Configure automated dependency update bots like [Renovate](https://docs.renovatebot.com/) or
> [Dependabot](https://dependabot.com/) with your source control platform (e.g., GitHub). These bots can automatically

Reviewer: ```suggestion
Configure automated dependency update bots like [Renovate](https://docs.renovatebot.com) or
[Dependabot](https://dependabot.com) with your source control platform (e.g., GitHub). These bots can automatically
```

Author: With the slash is technically correct, since without the slash is missing the path portion of the URL, but both render as URLs in the html and the browser makes it work the same.  I am committing this change given it does look better to the eyes.  

**devtools shows the appended slash**

<img width="1147" height="258" alt="image" src="https://github.com/user-attachments/assets/e721e54f-925f-4f3e-827f-0a369247afab" />

 **chrome hides the slash in the URL bar too**

<img width="1543" height="168" alt="image" src="https://github.com/user-attachments/assets/a07d68fa-7b98-426c-9944-b92dfe9cc6d8" />
